cmake/common.cmake

```diff
@@ -571,7 +571,7 @@ set(CCF_NETWORK_TEST_ARGS
   ${PBFT_ARG}
 )
 
-# Lua generic app
+# SNIPPET: Lua generic application
```

Comment:
The logging example was a little confusing because of the `stub_for_code_signing.cpp` inclusion.